tools/esp32/Bootloader.mk

```diff
@@ -0,0 +1,123 @@
+############################################################################
+# tools/esp32/Bootloader.mk
```

Comment:
Since this makefile generate the binary for target, not for PC, should we put it to arch/chip folder?

Reply:
Sure, it makes sense. Moved it to `arch/xtensa/src/esp32/`.
Thanks.